### PR TITLE
Rename file

### PR DESCRIPTION
**Pull request by guillermoherrero**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/guillermoherrero/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ThisNewFile [hunk 1]
```diff
@@ -1 +0,0 @@
-This is just a new file to test the Code Ninja functionalities.
```
Comment (guillermoherrero) [inline ThisNewFile]: was there really a change ?

Comment (guillermoherrero) [inline ThisNewFile]: I thought I put a 2 there

Comment (guillermoherrero) [inline ThisNewFile]: Well, I guess i did it the other way around !completed  !resolve

**Comment by guillermoherrero** [comment]
Wrong merging direction

**Pull request closed without merging**